src/styles/variables.css

```diff
@@ -242,4 +244,21 @@
   --type-sm: 14px;
   --type-xs: 12px;
   --type-xxs: 10px;
+
+  --KInputBackground: var(--section_colors-tertiary);
```

Comment:
Please do no use and overwrite kongponents variables in `variables.css`

If needed - please create a new variable and set value to corresponding value